server/repository/src/migrations/v2_05_00/diagnosis_add_to_invoice.rs

```diff
@@ -15,14 +15,6 @@ impl MigrationFragment for Migrate {
         "#
         )?;
 
```

Comment:
```suggestion
// We don't re-translate existing invoices here, as they could have been changed since the sync buffer record and could be overwritten
```